### ISSUE TITLE
Will this work on non-jobposting/broadcastevent posts?

### ISSUE THREAD

**Issue by hithismani** [goenning/google-indexing-script#74]
Thanks a lot for creating this!

During setup, I noticed the [Indexing API's quickstart guide](https://developers.google.com/search/apis/indexing-api/v3/quickstart) mentions this: 

> The Indexing API can only be used to crawl pages with either [JobPosting](https://developers.google.com/search/docs/appearance/structured-data/job-posting) or [BroadcastEvent embedded in a VideoObject](https://developers.google.com/search/docs/appearance/structured-data/video#broadcast-event).

![image](https://github.com/user-attachments/assets/01b9a982-b29b-46c7-b5b7-f3748605389e)

The reason I'm asking, is because `https://indexing.googleapis.com/v3/urlNotifications:publish` always returns the below:
```
{
    "urlNotificationMetadata": {
        "url": "<url submitted>"
    }
}
```
While https://indexing.googleapis.com/v3/urlNotifications/metadata?url=<above url, encoded> always returns:

```
{
    "error": {
        "code": 404,
        "message": "Requested entity was not found.",
        "status": "NOT_FOUND"
    }
}
```

Basically, been noticing that these api endpoints don't work without [proper approval now](https://developers.google.com/search/apis/indexing-api/v3/quota-pricing#request-quota?authuser=1).

Here's a link: to the [Support Form](https://support.google.com/webmasters/thread/298350253/index-api-receives-url-update-publish-request-but-errors-on-getting-status?hl=en) page on this, and the official API. 

Just thought I'd ask! As [the form they link us to](https://docs.google.com/forms/d/e/1FAIpQLSc_mpLw3WnnCt3pVbUHYZZ6ZdOS-c0GIj-WZ_k54SG-jDqCXQ/viewform), doesn't mention access to API for anything apart from Job Posting and Live Streaming.  Which could then indicate that we need to update this on the ReadMe as well.

**Comment by goenning** [comment]
It used to, but it seems like that changed recently. I'll update the readme.
